R/data.R

```diff
@@ -417,3 +417,20 @@ NULL
 #' }
 #' @source \url {https://phw.nhs.wales/services-and-teams/child-measurement-programme/cmp-2022-23/}
 "hl_reception_overweight_obese"
+
+#' Wales rough sleepers - (2023)
+#'
+#' A dataset containing statistics on rates of rough sleepers by 1,000, by
+#' Local Authority
+#'
+#' @format A data frame with 22 rows and 3 variables:
+#' \describe{
+#' \item{ltla22_name}{Local Authority Code}
+#' \item{rough_sleepers_per_1000}{People assessed as rough sleepers per 1,000
```

Comment:
Please specify in the description the date range of the rough sleepers data (including month) and note the date range of the population data.